analytics_data_api/management/commands/create_elasticsearch_learners_indices.py

```diff
@@ -0,0 +1,84 @@
+from elasticsearch import Elasticsearch
```

Comment:
According to our [python guidelines](https://github.com/edx/edx-platform/wiki/Python-Guidelines#imports-order), I think this is correct.